salesforce_api/kafka_salesforce_integration/kafka-salesforce-pricebook_update/sample.bal

```diff
@@ -0,0 +1,44 @@
+import ballerinax/kafka;
```

Comment:
Format the file using ballerina formatter

Reply:
Updated in [bcf09ac](https://github.com/ballerina-guides/integration-samples/pull/31/commits/bcf09ac42e87543485854586312facacacd050cb)